package.json

```diff
@@ -27,13 +27,15 @@
     "@alwaysmeticulous/cli": "^2.25.2",
     "@alwaysmeticulous/common": "^2.25.0",
     "@alwaysmeticulous/replayer": "^2.25.0",
+    "lodash-es": "^4.17.21",
```

Comment:
We could do `https://www.npmjs.com/package/lodash.debounce` so that yarn install & compile is a bit faster, and docker image is a bit smaller?

Reply:
(since we're not just shipping the pre-compiled bundle tree-shaking won't help us)